static/css/restyle/restyle.less

```diff
@@ -2121,6 +2121,10 @@ h3.author .transfer-ownership {
     float: left;
     margin-left: 0;
     width: 74%;
+
+    .html-rtl& {
+      margin-right: 0;
```

Comment:
This looks like it's not an inversion but there's a 210px right margin happening elsewhere.